Wrong examples without required options for `dotnet store`

The Examples section contains wrong code
```
dotnet store --manifest packages.csproj --framework-version 2.0.0
```
and the second one
```
dotnet store --manifest packages.csproj --skip-optimization
```
returns the same error

>  error MSB4057: The target "ComposeStore" does not exist in the project.

It is because `dotnet store` has required options: `-f` and `-r`

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fc238c72-c1eb-a09f-5b7b-801722924d38
* Version Independent ID: dbfcef0a-1079-0a2c-6bda-d2b57445ec29
* Content: [dotnet store command - .NET CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-store)
* Content Source: [docs/core/tools/dotnet-store.md](https://github.com/dotnet/docs/blob/main/docs/core/tools/dotnet-store.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

--- a/docs/core/tools/dotnet-store.md
+++ b/docs/core/tools/dotnet-store.md
@@ -74,13 +74,13 @@ dotnet store -h|--help
 - Store the packages specified in the *packages.csproj* project file for .NET Core 2.0.0:
 
   ```dotnetcli
-  dotnet store --manifest packages.csproj --framework-version 2.0.0
+  dotnet store --manifest packages.csproj --framework-version 6.0.1 --framework net6.0 --runtime win-x64
   ```
 
 - Store the packages specified in the *packages.csproj* without optimization:
 
   ```dotnetcli
-  dotnet store --manifest packages.csproj --skip-optimization
+  dotnet store --manifest packages.csproj --skip-optimization --framework net6.0 --runtime linux-x64
   ```
 
 ## See also

--- a/docs/core/tools/dotnet-store.md
+++ b/docs/core/tools/dotnet-store.md
@@ -57,7 +57,7 @@ dotnet store -h|--help
 
 - **`--skip-optimization`**
 
-  Skips the optimization phase.
+  Skips the optimization phase. For information about optimization, see [Preparing a runtime environment](../deploying/runtime-store.md#preparing-a-runtime-environment).
 
 - **`--skip-symbols`**
 

--- a/docs/core/tools/dotnet-store.md
+++ b/docs/core/tools/dotnet-store.md
@@ -57,7 +57,7 @@ dotnet store -h|--help
 
 - **`--skip-optimization`**
 
-  Skips the optimization phase. For information about optimization, see [Preparing a runtime environment](../deploying/runtime-store.md#preparing-a-runtime-environment).
+  Skips the optimization phase. For more information about optimization, see [Preparing a runtime environment](../deploying/runtime-store.md#preparing-a-runtime-environment).
 
 - **`--skip-symbols`**
 
@@ -71,7 +71,7 @@ dotnet store -h|--help
 
 ## Examples
 
-- Store the packages specified in the *packages.csproj* project file for .NET Core 2.0.0:
+- Store the packages specified in the *packages.csproj* project file for .NET 6.0.1:
 
   ```dotnetcli
   dotnet store --manifest packages.csproj --framework-version 6.0.1 --framework net6.0 --runtime win-x64

